dotnet/DotnetModuleSample/DotnetModuleSample/DotnetModuleSample.csproj.user

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You only need to update the documentation to let user to add the start argument.